assets/css/helper.scss

```diff
@@ -171,6 +171,156 @@ $color_button_secondary: $woo_pink2;
 }
 
 /*------------------------------------------------------------------------------
+  Subscriptons Header
+------------------------------------------------------------------------------*/
+
+.wc-helper {
+	.subscriptions-header{
```

Comment:
We need apply some coding standards for CSS, getting really messy with a lot of different styles, like here all CSS without space before `{`, something that you don't see in any of the others 15 SCSS files in this project.